carball/json_parser/actor/base.py

```diff
@@ -0,0 +1,40 @@
+COMPONENT_REPLICATED_ACTIVE_KEY = 'TAGame.CarComponent_TA:ReplicatedActive'
+REPLICATED_RB_STATE_KEY = 'TAGame.RBActor_TA:ReplicatedRBState'
+COMPONENT_ACTIVE_KEY = 'TAGame.CarComponent_TA:Active'
+
+
+class BaseActorHandler(object):
+    """
+    Base class for actor handlers.
+
+    If can_handle is not overridden, it will check the type_name attribute. The priority attribute determines the order
+    of the handlers. Lower number means higher priority. If the priority is the same, the order will depend on the order
+    of actors int he replay.
```

Comment:
in the* replay

also I'd put the comments for the two attributes on different lines.